requirements.txt

```diff
@@ -6,3 +6,4 @@ pandas==2.2.2
 sentence-transformers==3.0.1
 tiktoken==0.7.0
 tqdm==4.66.4
+lxml==5.3.0
```

Comment:
Can remove if we go with the decision to make it optional

Reply:
Done